main/src/main/java/net/citizensnpcs/npc/CitizensTraitFactory.java

```diff
@@ -82,6 +83,7 @@ public CitizensTraitFactory(Citizens plugin) {
         registerTrait(TraitInfo.create(AttributeTrait.class));
         registerTrait(TraitInfo.create(Anchors.class));
         registerTrait(TraitInfo.create(BatTrait.class));
+        registerTrait(TraitInfo.create(BeTargedByTrait.class));

```

Comment:
Can we name it "TrackTargetsTrait"

Reply:
We are being targeted by others instead of targeting others. IDK if this is a good name.